px-map-tile-layer-google.html

```diff
@@ -0,0 +1,140 @@
+<!--
+    Relative paths assume component is being run from inside an app or another component, where dependencies are flat
+    siblings. When this component is run from its own repo (e.g. tests, examples), we assume the server is started with
+    'gulp serve' (or similar server setup) to enable correct finding of bower dependencies for local runs.
+-->
+<link rel="import" href="../polymer/polymer.html"/>
+
+<!-- Load external dependencies -->
+<script src="../leaflet-google-tiles/leaflet-google-layer.js"></script>
+
+<!-- Load required PxMapBehaviors -->
+<link rel="import" href="px-map-behavior-tile-layer.html">
+
+<!--
+Adds a Google Maps Tile layer. Google tiles are requested by authenticating with a
+valid Google Maps Tile API key and choosing the type of imagery to display.
+
+Note that a Google Maps Tile API key is not provided by this component. You will need
+to [obtain your own API key](https://developers.google.com/maps/documentation/tile/#api-key).
+
+If you do not provide an API key, the map will not load any tiles.
+
+### Usage
+
+    <px-map>
+      <px-map-tile-layer-google key="XXXXXXXXXXXXXXXXXXXXX"></px-map-tile-layer-google>
+    </px-map>
+
+@element px-map-tile-layer-google
+@blurb Loads base layer tiles from the Google Maps API
+@homepage index.html
+@demo index.html
+-->
+
+<dom-module id="px-map-tile-layer-google">
+  <style>
+    :host { display: none }
+  </style>
+  <template></template>
+</dom-module>
+<script>
+  Polymer({
+    is: 'px-map-tile-layer-google',
+    behaviors: [PxMapBehavior.Layer],
+
+    properties: {
+      /**
+       * A valid Google Maps Tile API key. Required to fetch the tiles - if it is not
+       * provided, no tiles will be loaded.
+       */
+      key: {
+        type: String,
+        observer: 'shouldUpdateInst'
+      },
+
+      /**
+       * The map type to use. Choose from the following options:
+       *
+       * - 'roadmap' - (default) The standard Google Maps painted map tiles.
+       * - 'satellite' - Satellite imagery.
+       *
+       * The following Google imagery sets are not supported: 'terrain, streetview'
+       */
+      imagery: {
+        type: String,
+        value: 'roadmap',
+        observer: 'shouldUpdateInst'
+      },
+
+      /**
+       * A region identifier representing the physical location of the user to whom
+       * the tiles are shown.
+       * Accepts Unicode region subtag identifiers which (generally) have a one-to-one
+       * mapping to country code Top-Level Domains (ccTLDs).
+       */
+      region: {
+        type: String,
+        value: 'us',
+        observer: 'shouldUpdateInst'
+      },
+
+      /**
+       * A Common Locale Data Repository language identifier indicating the
```

Comment:
Could you link add a link to a good webpage with the common locale data identifiers? Shortcut for users who don't know them off the top of their head and will pop open a new tab to google for it.